packages/block-editor/src/components/use-editor-feature/index.js

```diff
@@ -18,11 +18,11 @@ const deprecatedFlags = {
 		settings.disableCustomColors === undefined
 			? undefined
 			: ! settings.disableCustomColors,
-	'gradient.custom': ( settings ) =>
+	'color.customGradient': ( settings ) =>
 		settings.disableCustomGradients === undefined
 			? undefined
 			: ! settings.disableCustomGradients,
-	'fontSize.custom': ( settings ) =>
+	'typography.customFontSize': ( settings ) =>
```

Comment:
Just below this statement we're doing `lineheight.custom` but it looks like we should do `typography.customLineHeight` instead, right? (as well as the appropriate change in the line-height hook)

Reply:
Nice catch it was fixed 👍